app.js

```diff
@@ -121,14 +134,45 @@ router.post('/contract/view/:name/:methodName', async ctx => {
 
 router.get('/account/:name', async ctx => {
     const response = await client.request('view_account', [{
-        account_id: await hash(ctx.params.name),
+        account_id: await encodeAccountNameForRpc(ctx.params.name),
         method_name: '',
         args: []
     }]);
     checkError(ctx, response);
     ctx.body = response.result;
 });
 
+/**
+ * Create a new account. Generate a throw away account id (UUID).
+ */
+router.post('/account', async ctx => {
+    // TODO: this is using alice account to create all accounts. We may want to change that.
+    const nonce = await getNonce(ctx, defaultSender);
+    ensureTrue(nonce);
+    const newAccountName = uuidV4();
```

Comment:
This is totally not blocking this PR, but I think it'll be nice to have readable word combinations as account names instead, using e.g. using this package:
https://www.npmjs.com/package/rword